docker-parse/azuredeploy.json

```diff
@@ -0,0 +1,273 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed."
+      }
+    },
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "The location where the Virtual Machine will be deployed."
+      },
+      "allowedValues": [
+        "West US",
+        "East US",
+        "Southeast Asia",
+        "East Asia",
+        "West Europe"
+      ]
+    },
+    "parseAppId": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The application id to host with this server instance."
+      }
+    },
+    "parseMasterKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The master key to use for overriding ACL security."
+      }
+    },
+    "parseFileKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "For migrated apps, this is necessary to provide access to files already hosted on Parse."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    }
+  },
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "ubuntuOSVersion": "15.04",
+    "OSDiskName": "osdiskfordockersimple",
+    "nsgName": "myNSG",
+    "nicName": "myVMNic",
+    "extensionName": "DockerExtension",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "storageAccountType": "Standard_LRS",
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+    "vmStorageAccountContainerName": "vhds",
+    "vmName": "MyDockerVM",
+    "vmSize": "Standard_D1",
```

Comment:
Is the migrated app going to fine with a D1? Maybe offer this as a parameter with some more sizes as allowed values?